learning/katas/go/Introduction/Hello Beam/Hello Beam/task-info.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+type: edu
+files:
+- name: test/task_test.go
+  visible: false
+- name: test/output.txt
+  visible: false
+- name: task.go
+  visible: true
+  placeholders:
+  - offset: 1143
+    length: 28
+    placeholder_text: 'TODO: inject your pipeline with "Hello Beam".'
```

Comment:
In Java & Python katas, we only use "TODO()".
I find that the kata description and hint(s) provide good enough instruction.
Do you think this will be more useful for the learner?